sandbox/samples/cpp/sphereflake/sphereflake.cpp

```diff
@@ -0,0 +1,157 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2017 Esteban Tovagliari, The appleseedhq Organization
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// appleseed.renderer headers.
+#include "renderer/api/log.h"
+#include "renderer/api/object.h"
+#include "renderer/api/scene.h"
+#include "renderer/api/project.h"
+
+// appleseed.foundation headers.
+#include "foundation/math/vector.h"
+#include "foundation/utility/string.h"
+
+// Standard headers.
+#include <cstring>
+#include <memory>
+
+// Sphereflake headers.
+#include"sphereflakedata.h"
```

Comment:
Add space after `#include`.

Reply:
Typically the sphere snowflake is generated procedurally no? :)

Reply:
We could, but the original code from http://www.realtimerendering.com/resources/SPD/ is pretty old and I thought it was too much work to modernize it just for a sample.
I was surprised it even compiles now (with some minor fixes).